packages/css/src/transformCss.ts

```diff
@@ -220,6 +216,10 @@ class Stylesheet {
     });
   }
 
+  transformProperties(cssRule: CSSPropertiesWithVars) {
```

Comment:
Should this be `CSSRule` instead?

Reply:
I thought so too, but type checking fails if we use `CSSRule` here. I made it `CSSPropertiesWithVars` as that's the input type for both `transformContent` and `pixelifyProperties`.

Reply:
Oh its because we pass it `CSSRule['rule']` from the call sites. I see now.